Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM adoptopenjdk/openjdk9:x86_64-ubuntu-jdk-9.0.4.11-slim
+
+ENV JAVA_TOOL_OPTIONS=""
+
+RUN apt-get update && apt-get install -y maven git curl unzip zip
+
+RUN curl -s http://get.sdkman.io | bash
+
+ARG JDK1
+RUN test -n "$JDK1" || (echo "JDK1 build-arg not provided - choose one from:" && bash -c "source $HOME/.sdkman/bin/sdkman-init.sh && sdk l java" && false)
```

Comment:
How about getting the JDKs via a volume from the host? This would avoid the repeated download for building version-specific images. WDYT?

Reply:
So, my assumption was that if someone wants to just try it, that would be handy to have that fully automated. 

If someone wants to pass volumes from the host, they can just simply run the code without using the easy solution - but naturally, that could be added as well

Reply:
Notice how easy CI becomes thanks to that: 
https://github.com/AdoptOpenJDK/jdk-api-diff/blob/0292872fcfb7705eb5d648cd16be7d7137337ff2/.github/workflows/build.yml